### PR TITLE
Add console_shortcut on windows.

### PR DESCRIPTION
**Pull request by ericpre**
Add `console_shortcut` package as discussed in https://github.com/conda-forge/miniforge/issues/62.

A few comments:
- This shortcut follow the Anaconda convention by creating a shortcut where in the same folder as the Anaconda distribution does - with the [`console_shortcut` package](https://anaconda.org/anaconda/console_shortcut).
- Since this recipe doesn't package any code, only contains the icon and the json files, I don't think that there is a need to specify and include a license.
- The icon used for the start menu is in public domain (https://github.com/paomedia/small-n-flat)

Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/console_shortcut/console_shortcut.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "menu_name": "Anaconda${PY_VER} ${PLATFORM}",
```
Comment (isuruf) [inline recipes/console_shortcut/console_shortcut.json:2]:
```suggestion
    "menu_name": "Miniforge${PY_VER} ${PLATFORM}",
```

File: recipes/console_shortcut/console_shortcut.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "menu_items":
+    [
+        {
+            "name": "Anaconda Prompt",
```
Comment (isuruf) [inline recipes/console_shortcut/console_shortcut.json:6]:
```suggestion
            "name": "Miniforge Prompt",
```

File: recipes/console_shortcut/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{% set version = "1.0" %}
+{% set build = 0 %}
+
+package:
+  name: console_shortcut
```
Comment (isuruf) [inline recipes/console_shortcut/meta.yaml:5]:
```suggestion
  name: miniforge_console_shortcut
```

File: recipes/console_shortcut/console_shortcut.json (hunk 1 above)
Comment (ericpre) [reply recipes/console_shortcut/console_shortcut.json]: Ideally, we would like to use the name of the distribution, is this information available somewhere?

Comment (ericpre) [reply recipes/console_shortcut/console_shortcut.json]: > Ideally, we would like to use the name of the distribution, is this information available somewhere?

I would suggest to use `Miniforge` for now and when there is a better solution to improve it in the feedstock.

File: recipes/console_shortcut/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  number: {{ build }}
+  skip: True  # [not win]
+
+test:
+  commands:
+    - if not exist %PREFIX%\\Menu\\console_shortcut.json exit 1
+    - if not exist %PREFIX%\\Menu\\console_shortcut.ico exit
+
+about:
+  summary: Command prompt shortcut for Windows with base environment activated
```
Comment (isuruf) [inline recipes/console_shortcut/meta.yaml:18]:
```suggestion
  home: https://github.com/conda-forge/console_shortcut-feedstock
  summary: Command prompt shortcut for Windows with base environment activated
  license: BSD-3-Clause
  license_file: LICENSE.txt
```

for the license add https://github.com/AnacondaRecipes/aggregate/blob/master/LICENSE.txt to `recipe/console_shortcut/LICENSE.txt`

**Comment by ericpre** [comment]
Thanks @isuruf!

**Comment by isuruf** [comment]
Thanks @ericpre

**Pull request merged**